HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -403,6 +403,11 @@ fatal.deprecated_java_version=HMCL will require Java 11 or later to run in the f
   It is recommended to install the latest version of Java to ensure that HMCL works properly.\n\
   \n\
   You can continue to keep the old version of Java. HMCL can recognize and manage multiple Java versions and will automatically select the appropriate Java for you based on the game version.
+fatal.deprecated_java_version.update=HMCL will require Java 11 or later to run in the future. Please install the latest version of Java to ensure that HMCL can complete the upgrade.\n\
+  \n\
+  You can continue to keep the old version of Java.\
+  HMCL can recognize and manage multiple Java versions\
+  and will automatically select the appropriate Java for you based on the game version.
```

Need to add a half-width space after `HMCL can recognize and manage multiple Java versions`, otherwise it will appear as `versionsand`.

```suggestion
fatal.deprecated_java_version.update=HMCL will require Java 11 or later to run in the future. Please install the latest version of Java to ensure that HMCL can complete the upgrade.\n\
  \n\
  You can continue to keep the old version of Java.\
  HMCL can recognize and manage multiple Java versions \
  and will automatically select the appropriate Java for you based on the game version.
```